news/2019-02-04-community-choice-2018-results.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Community Choice 2018: Results"
+date: 2019-02-04 10:30:00 +0000
+---
+
+You've had your say, and we have our winners! Check out the new Community Favourites of 2018, featuring some of the closest voting we've ever seen!
+
+[![](/wiki/shared/news/2019-01-17-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)
+
+### osu!
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/kda.jpg)](https://osu.ppy.sh/beatmapsets/874948)
+
+**2489** voters could feel the rush now as they enjoyed [Nathan](https://osu.ppy.sh/users/4785223)'s take on the video-game cross k-pop masterpiece that took the world by storm, **[K/DA - POP/STARS (ft. Madison Beer, (G)I-DLE, Jaira Burns)](https://osu.ppy.sh/beatmapsets/874948)**!
+
+This will likely come as a surprise to some who expected one map from the ubiquitous deluge of 'farming' maps to nab top place, but as this goes to show, sometimes you just need to stop farming creeps and get out there and start nabbing kills, something that Nathan and his collaborators managed with effortless ease.
+
+The finalists were:
+
+[Panda Eyes - ILY by M a r v o l l o](https://osu.ppy.sh/beatmapsets/653534) with **1935** votes total.
+
+[Vickeblanka - Black Rover (TV Size) by Sotarks](https://osu.ppy.sh/beatmapsets/781509) with **1911** votes total.
+
+[DECO*27 - HIBANA feat. Hatsune Miku by Pho](https://osu.ppy.sh/beatmapsets/671199) with **1724** votes total.
+
+[SOOOO - Happppy song by Kuron-kun](https://osu.ppy.sh/beatmapsets/744593) with **1713** votes total.
+
+[Aitsuki Nakuru - Monochrome Butterfly by Sotarks](https://osu.ppy.sh/beatmapsets/770300) with **1679** votes total.
+
+[Omoi - Teo by Kroytz](https://osu.ppy.sh/beatmapsets/721804) with **1572** votes total.
+
+[Camellia feat. Nanahira - Bassdrop Freaks (2018 "Redrop" ver.) by Mir](https://osu.ppy.sh/beatmapsets/806859) with **1569** votes total.
+
+[Camellia - Exit This Earth's Atomosphere (Camellia's ''PLANETARY//200STEP'' Remix) by ProfessionalBox](https://osu.ppy.sh/beatmapsets/855677) with **1475** votes total.
+
+[YUC'e - Future Candy by Nathan](https://osu.ppy.sh/beatmapsets/546820) with **1388** votes total.
+
+[Check out the full results over on the contest listing for this mode!](https://osu.ppy.sh/community/contests/74)
+
+### osu!taiko
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/xinobikill.jpg)](https://osu.ppy.sh/beatmapsets/742966)
+
+**121** drum fanatics stepped forward and said "hell yes" to [SOUND HOLIC feat. Nana Takahashi - XINOBIKILL](https://osu.ppy.sh/beatmapsets/742966) by [Skull Kid](https://osu.ppy.sh/users/3044264), in some of the closest voting we've ever observed in a Community Choice poll *or* a "Best Of" to date.
```

Comment:
Map link should be bold like osu's number one? Same goes for catch/mania. 
Makes the winning map name easier to notice.

Reply:
done